app/models/event.rb

```diff
@@ -0,0 +1,24 @@
+class Event < ApplicationRecord
+  belongs_to :move
+
+  enum event_types: {
```

Comment:
it looks like these are more `event_names`

Reply:
yep, I think `event_names` is a better term, I'll rename it 👍